Formula/p/proto.rb

```diff
@@ -0,0 +1,48 @@
+class Proto < Formula
+  desc "Pluggable multi-language version manager"
+  homepage "https://moonrepo.dev/proto"
+  url "https://github.com/moonrepo/proto/archive/refs/tags/v0.30.0.tar.gz"
+  sha256 "a564e4c8a4d2483b19e02bd1b3cdac5db4ad61cc0e2d217203f0a641897ba7f4"
+  license "MIT"
+  head "https://github.com/moonrepo/proto.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
```

Comment:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build

  on_linux do
    depends_on "xz"
  end
```

Reply:
`peg-config` is in the `rust` dep tree, so we don't need to place it in the conditional.

There's no linkage with `liblzma` on macOS, so we can just set `xz` as a dependent on Linux.

Reply:
Thanks for the help here, this was the part I was most confused about.